tests/posix_semaphore/main.c

```diff
@@ -260,13 +266,20 @@ void test4(void)
             puts("first: timed out");
         }
     }
-    stop = xtimer_now_usec64() - start;
-    if ((stop < (exp - 100)) || (stop > (exp + 100))) {
-        fmt_u64_dec(uint64_str, stop);
+
+    fmt_u64_dec(uint64_str, elapsed);
+    if (elapsed < (exp - 100)) {
         printf("first: waited only %s usec => FAILED\n", uint64_str);
     }
+#ifdef BOARD_NATIVE
```

Comment:
Please define the literal somewhere "upstairs" and use that define in the logic.

Reply:
Sure.